core/src/main/java/io/grpc/internal/CensusStatsModule.java

```diff
@@ -71,6 +71,15 @@
   private static final Logger logger = Logger.getLogger(CensusStatsModule.class.getName());
   private static final double NANOS_PER_MILLI = TimeUnit.MILLISECONDS.toNanos(1);
 
+  static final boolean OPENCENSUS_IMPL_PRESENT;
+
+  static {
+    ClassLoader loader = io.opencensus.stats.StatsComponent.class.getClassLoader();
```

Comment:
I'm wondering if this can be `CensusStatsModule.class.getClassLoader()`, and people who don't want census can remove runtime dependency of census completely, namely making census api as a compile-time-only dependency for grpc library.

Reply:
@dapengzhang0 you mean also remove the existing `opencensus-api` dependency? I don't think that would be possible without the options discussed in #5110 which both require some kind of breakage, at a minimum would need existing census users to amend their builds.

Reply:
I mean not removing opencensus-api transparent dependency from grpc library, but about the ability for people who don't want census to explicitly remove that dependency in their own project.

Reply:
oh I see what you mean, yes I think that should be possible, it would require a couple more changes than this though, for example `io.opencensus.trace.Tracing` is also referenced directly in the channel and server builder classes.